controllers/apply/awards-for-all/form.js

```diff
@@ -908,20 +994,34 @@ module.exports = function({
                 2 St James’ Gate, Newcastle upon Tyne, NE1 4BE.
              </p>
              <p>When you click submit the Terms and Conditions as agreed to above become binding.</p>`,
-            cy: ''
+            cy: `<p>
+                Rydym yn gwybod bod y rhan fwyaf o’r miloedd sy’n 
+                chwilio am arian gennym yn ddiffuant. Er hynny, rydym 
+                weithiau yn derbyn ceisiadau twyllodrus ac felly mae gennym 
+                ddyletswydd i wneud gwiriadau ar unigolion mewn sefydliadau 
+                sy’n ymgeisio am grantiau. Bydd y wybodaeth bersonol rydym wedi’i 
+                gasglu gennych felly yn cael ei rannu gydag asiantaethau atal twyll, 
+                a fydd yn ei ddefnyddio i atal twyll a gwyngalchu arian, 
+                ac i wireddu eich hunaniaeth. Os oes twyll yn cael ei ganfod, 
+                gellid eich gwrthod rhag rhai gwasanaethau, arian neu gyflogaeth.
+            </p>
+            <p>
+                Gall gwybodaeth pellach ar sut bydd eich gwybodaeth yn cael ei ddefnyddio gennym a’r 
+                asiantaethau atal twyll hyn, eich hawliau gwarchod data a sut i gysylltu â ni, ei ganfod yn ein 
+                Hysbysiad Diogelu Data a Phreifatrwydd llawn, sydd wedi’i gyhoeddi ar ein gwefan 
+                <a href="/welsh/about/customer-service/data-protection">https://www.tnlcommunityfund.org.uk/welsh/about/customer-service/data-protection</a>. 
+                Cysylltwch â ni i ofyn am gopi caled drwy ffonio ein llinell gynghori ar 0345 4 10 20 30, 
```

Comment:
Highlighting the hyperlink change to make sure it exists.

Reply:
Yep this one is right!